File: LoginPassword.cpp
#include<iostream>
#include<fstream> // header file for file handling 
#include<string>  // for string values 

using namespace std;

int main(){ // main function

    string  appname , name, password;  // Declare String's
    string  app , n, p;
    
    cout<<"================================="<<endl;
    cout<<"           Create Account        "<<endl;
    cout<<"================================="<<endl;
    cout<<"Enter Application Name:"<<endl;
    getline(cin, appname);
    cout<<"Enter Username:"<<endl;
    getline(cin, name);                // using getline function to get input from user only for string
    cout<<"Enter Password:"<<endl;
    getline(cin, password);

    cout<<"================================="<<endl;
    cout<<"           Login Account         "<<endl;
    cout<<"================================="<<endl;
    cout<<"Enter Application Name:"<<endl;
    getline(cin ,app);
    cout<<"Enter Username:"<<endl;
    getline(cin, n);
    cout<<"Enter Password:"<<endl;
    getline(cin, p);

    if(n == name && p == password && app == appname){       // Program Logic
        cout<<"Login Successfully!"<<endl;
    } else {
        cout<<"Invalid Credentials!"<<endl;
    }

    ofstream file("Login_Saved.txt", ios::app);  // Use file hanldin to saved login passwords
    
    if(file.is_open()) {

        file<<"================================="<<endl;
        file<<"         Password Manager        "<<endl;
        file<<"================================="<<endl;
        file<<"Application Name:\n"<<app<<endl;
        file<<"Username:\n"<<n<<endl;  
        file<<"Password:\n"<<p<<endl; 
        file.close();
    } else {
        cout<<"Unable to open file!"<<endl;
    }

    return 0;
}

File: LoginPassword.py
print("===================================") 
print("           Create Account          ")
print("===================================")

name = input("Enter Username:\n")
password = input("Enter Passowrd:\n")

print("===================================")
print("           Login  Account          ")
print("===================================")

n = input("Enter Username:\n")
p = input("Enter Passowrd:\n")


if n == name and p == password: # Program Logic
    print("Login Sucessfully!")
else:
    print("Invalid Crediantials!")    

file = open("login_saved.txt" , "w") # Use File Handling to saved passwords
file.write("===================================\n")
file.write("            Saved Passwords        \n")
file.write("===================================\n")
file.write("Username:\n"+n+"\n")
file.write("Password:\n"+p+"\n")

file.close()